kumascript/src/api/web.ts

```diff
@@ -164,7 +173,7 @@ const web = {
       flawAttribute = ` data-flaw-src="${util.htmlEscape(flaw.macroSource)}"`;
     }
     // Let's get a potentially localized title for when the document is missing.
-    const titleWhenMissing = this.mdn.getLocalString(
+    const titleWhenMissing = (this.mdn as any).getLocalString(
```

Comment:
This is similar to the previous issue, except that `this.web.getJSONData()` works here (although `this.web` is the self-reference here) whereas `this.mdn.getLocalString()` doesn't. 😕